thunder/executors/nvfuserex_impl.py

```diff
@@ -3188,6 +3188,35 @@ def argsort_transform(
 # Register argsort with NVFuser
 register_supported(prims.argsort, argsort_transform, _argsort_check_)
 
+
+def _grouped_mm_check_(
+    a: TensorProxy,
+    b: TensorProxy,
+    offs: TensorProxy,
+    bias: TensorProxy = None,
+    out_dtype=None,
+) -> bool:
+    return True
```

Comment:
I guess it'd be better to have nvfuser version check and `are_supported_tensors` here 

https://github.com/Lightning-AI/lightning-thunder/blob/dbf6bad30cf38606f2feaec629f19357bba0b74d/thunder/executors/nvfuserex_impl.py#L222